telemetry/specs/INT.mdk

```diff
@@ -1003,7 +1185,7 @@ INT Metadata Header and Metadata Stack, followed by TCP payload:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   | Ver=1 | 0 |0|0|0|     Reserved      | HopML=2 |RemainingHopC=6|
+   | Ver=1 | 0 |0|0|0|0|   Reserved      | HopML=2 |RemainingHopC=6|
```

Comment:
All these examples need the addition of DS ID, DS Flags, DS Field

Reply:
Should we add examples of Ver=2 instead?

Reply:
We will do this PR only at version 2. Hence no need to worry about version 1 at all.

Reply:
Agree. Removed Ver=1